app/components/tooltip.js

```diff
@@ -39,6 +40,13 @@ export default class ToolTip extends PureComponent {
         }
     };
 
+    componentDidUpdate(prevProps) {
+        if (prevProps.actions.length !== this.props.actions.length && getToolTipVisible()) {
```

Comment:
If `actions` is optional then calling length of undefined will throw an error.

Reply:
I think it should be required.